contrib/mpi-proxy-split/split_process.cpp

```diff
@@ -46,10 +46,7 @@
 #include "procmapsutils.h"
 #include "util.h"
 #include "dmtcp.h"
-
-#ifndef CENTOS
-# define CENTOS
-#endif
+#include "mtcp/mtcp_util.h"
```

Comment:
I don't think you need it?

Reply:
yes, i was using it before and i removed my changes. forgot to remove this one.